pkg/frontend/export.go

```diff
@@ -465,6 +465,7 @@ func constructByte(obj interface{}, bat *batch.Batch, index int32, ByteChan chan
 			case types.T_char, types.T_varchar, types.T_blob, types.T_text, types.T_binary, types.T_varbinary:
 				value := addEscapeToString(vec.GetBytesAt(i))
 				writeByte = appendBytes(writeByte, value, symbol[j], closeby, true)
+				//TODO: How should we export T_array in CSV? Is it as "[1,2,3]"?
```

Comment:
can the vector type be exported to string? missing here

Reply:
@flying-x could you kindly share your thoughts on this?